_includes/pbc/all/install-features/202304.0/install-the-product-feature.md

```diff
@@ -0,0 +1,401 @@
+
```

Comment:
please use the corresponding doc from 202212.0 as a reference and add the new functionality (the document will have different structure so I haven't reviewed this one in details). Recheck the new structure and place plugin and config according to a new structure to the proper section